services/cinema_hall.py

```diff
@@ -0,0 +1,21 @@
+from django.db.models import QuerySet
+
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:

```

Comment:
Indicate a `QuerySet` of what instances you return in the return type annotation